docs/guide/robots.md

```diff
@@ -5,54 +5,59 @@
-| ![icon.png](images/robots/khepera4/icon.png)    | [Khepera IV](khepera4.md)      | K-Team               | *Two-wheeled robot*             |
-| ![icon.png](images/robots/khr-2hv/icon.png)     | [KHR-2HV](khr-2hv.md)          | KONDO                | *Small size humanoid*           |
-| ![icon.png](images/robots/khr-3hv/icon.png)     | [KHR-3HV](khr-3hv.md)          | KONDO                | *Small size humanoid*           |
-| ![icon.png](images/robots/koala/icon.png)       | [Koala](koala.md)              | K-Team               | *Six-wheeled robot*             |
-| ![icon.png](images/robots/mantis/icon.png)      | [Mantis](mantis.md)            | micromagic           | *Hexapod*                       |
-| ![icon.png](images/robots/mavic-2-pro/icon.png) | [Mavic 2 PRO](mavic-2-pro.md)  | DJI                  | *Quadcopter drone*              |
-| ![icon.png](images/robots/microbot/icon.png)    | [Microbot](microbot.md)        | PICAXE               | *Two-wheeled robot*             |
-| ![icon.png](images/robots/mindstorms/icon.png)  | [Mindstorms](mindstorms.md)    | Lego                 | *Robotics kit*                  |
-| ![icon.png](images/robots/moose/icon.png)       | [Moose](moose.md)              | Clearpath Robotics   | *Eight-wheeled outdoor robot*   |
-| ![icon.png](images/robots/nao/icon.png)         | [Nao](nao.md)                  | SoftBankics          | *Small size humanoid*           |
-| ![icon.png](images/robots/pioneer2/icon.png)    | [Pioneer 2](pioneer2.md)       | Adept                | *Two-wheeled robot*             |
-| ![icon.png](images/robots/pioneer-3at/icon.png) | [Pioneer 3-AT](pioneer-3at.md) | Adept                | *Four-wheeled robot*            |
-| ![icon.png](images/robots/pioneer-3dx/icon.png) | [Pioneer 3-DX](pioneer-3dx.md) | Adept                | *Two-wheeled robot*             |
-| ![icon.png](images/robots/portal/icon.png)      | [Portal](portal.md)            | A4                   | *Sliding portal*                |
-| ![icon.png](images/robots/pr2/icon.png)         | [PR2](pr2.md)                  | Clearpath Robotics   | *Two arms on mobile a platform* |
-| ![icon.png](images/robots/puma/icon.png)        | [PUMA](puma.md)                | Unimation            | *Robotics arm*                  |
-| ![icon.png](images/robots/qrio/icon.png)        | [QRIO](qrio.md)                | Sony                 | *Small size humanoid*           |
-| ![icon.png](images/robots/robotis-op2/icon.png) | [Robotis OP2](robotis-op2.md)  | Robotis              | *Small size humanoid*           |
-| ![icon.png](images/robots/robotis-op3/icon.png) | [Robotis OP3](robotis-op3.md)  | Robotis              | *Small size humanoid*           |
-| ![icon.png](images/robots/salamander/icon.png)  | [Salamander](salamander.md)    | EPFL / BioRob        | *Bio-inspired robot*            |
```

Comment:
Is there a particular reason for changing the order of this item? the alphabetical order is not respected anymore.

Reply:
Obviously not, I think it comes when I updated the fork. Thank you for seeing this change that I missed.